docs/getstarted/index.md

```diff
@@ -36,13 +36,13 @@ If you have already downloaded and installed Docker, you are ready to run Docker
 
 You'll learn how to:
 
-* install Docker software for your platform
-* run a software image in a container
-* browse for an image on Docker Hub
-* create your own image and run it in a container
-* create a Docker Hub account and an image repository
-* create an image of your own
-* push your image to Docker Hub for others to use
+* Install Docker software for your platform
```

Comment:
these bullet points are the continuation of the sentence above:

You'll learn how to _install_ ... . They do not require capitalization.